### DISCUSSION TITLE
Problem with Icons in nvim-tree.lua Plugin

### DISCUSSION THREAD

**Discussion by kadeguilherme** [kovidgoyal/kitty#8101] [General]
Hello, I recently configured Neovim to use the nvim-tree.lua plugin, but when I launch Neovim, the icons don't appear. I read that it is not recommended to download Nerd Fonts, and I'm unsure how to properly configure this. Could someone guide me on how to set up the icons correctly? I’m still learning how to use Kitty and Neovim, so any help would be greatly!  Thank you !
![image](https://github.com/user-attachments/assets/2013e5d3-fc33-463d-b748-36e9f58029e9)
![image](https://github.com/user-attachments/assets/42952777-fe90-4bab-bbaa-7cb45075b42c)
![image](https://github.com/user-attachments/assets/38d71e77-2f19-490c-91b7-30fc7570bd43)

**Comment by kovidgoyal** [comment]
Make sure you are running up to date kitty (newer versions have nerd fonts builtin) and check if the nerd fonts are working outside of nvim using a printf or similar.
